cloud-api/src/main/java/com/sequenceiq/cloudbreak/cloud/CredentialConnector.java

```diff
@@ -4,24 +4,27 @@
 
 import javax.annotation.Nonnull;
 
+import org.apache.commons.lang3.StringUtils;
+
 import com.sequenceiq.cloudbreak.cloud.context.AuthenticatedContext;
 import com.sequenceiq.cloudbreak.cloud.context.CloudContext;
 import com.sequenceiq.cloudbreak.cloud.credential.CredentialNotifier;
 import com.sequenceiq.cloudbreak.cloud.model.CloudCredentialStatus;
 import com.sequenceiq.cloudbreak.cloud.model.ExtendedCloudCredential;
+import com.sequenceiq.cloudbreak.common.model.user.IdentityUser;
 
 /**
  * Manages the credentials e.g public keys on Cloud Plarform side
  */
-public interface CredentialConnector {
+public abstract class CredentialConnector {
```

Comment:
Please change back to interface and modify interactiveLogin method to default method